static/app/views/replays/detail/page.tsx

```diff
@@ -95,7 +96,7 @@ export default function Page({
         ) : (
           <Fragment>
             {isVideoReplay ? <FeedbackWidgetButton /> : <FeedbackButton />}
-            {isVideoReplay ? null : <ConfigureReplayCard />}
+            {isVideoReplay ? <ConfigureMobileReplayCard /> : <ConfigureReplayCard />}
```

Comment:
Looks like we have the replayRecord already is it's not still loading. Maybe we can pass in the platform and deep link to the specific docs for the project that the replay is inside; whether that's mobile or web specific. 

Maybe if the replayRecord is still loading then picks. Default value, or change links to somewhere else.. I'm thinking aloud here tho

Reply:
i thought about that but it's hard because the apple docs are structured differently in the URL than the react native or android. apple has an extra `.../apple/guides/ios/session-replay/` while android is `.../android/session-replay/` which means we have to case on the platform. it was already sort of a pain for the onboarding, and if we want to do it again for even more platforms then it could get complicated

plus if we cased on the platform for mobile then we'd have to do it for web, and there are a lot more web docs, esp the javascript sub-platforms. ex `.../javascript/guides/angular/session-replay/`

Reply:
Sounds like a Monday problem!

Reply:
discussed offline - we can't get the full platform from `replayRecord.platform` which makes casing on the platform difficult.

Reply:
one potential alternative is casing on the `sdk.name`